cmake/BundledAzureSdkCpp.cmake

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 function(use_bundled_libazure SOURCE_DIR BINARY_DIR)
-    set(PC "${Patch_EXECUTABLE}" -p1 -i "${SOURCE_DIR}/thirdparty/azure-sdk-cpp-for-cpp/azure-sdk-for-cpp-old-compiler.patch")
+    set(PC git reset --hard HEAD &&"${Patch_EXECUTABLE}" -p1 -i "${SOURCE_DIR}/thirdparty/azure-sdk-cpp-for-cpp/azure-sdk-for-cpp-old-compiler.patch")
```

Comment:
There is a missing space before `&&` and `"${Patch_EXECUTABLE}"`

Reply:
fixed in 0f7b060